README.md

```diff
@@ -28,8 +52,8 @@ Retrieve the Juju secrets list:
 user@ubuntu:~$ juju secrets --format=yaml
 ck0i0h3q457c7bgte4kg:
   revision: 1
-  owner: vault-k8s
-  label: vault-ca-certificate
```

Comment:
Not sure we want to delete this here, it really is a secret that will show up when we run `juju secrets`

Reply:
It's deleted as it's not our concern. In the juju secrets we could see more secrets such as below.  It's not written down because we do not use it in the next steps.
```
cks0akc24l7c77v23p2g:
  revision: 1
  expires: 2024-10-16T17:56:32Z
  owner: traefik-k8s/0
  label: |-
    afd8c2bccf834997afce12c2706d2ede------BEGIN CERTIFICATE REQUEST-----
    MIICqTCCAZECAQAwQjERMA8GA1UEAwwIMTAuMC4wLjQxLTArBgNVBC0MJGQzNWYw
    ZWYyLWYwYmQtNDVjYS1iNGY2LWY2NjQzMDlkMDRhMDCCASIwDQYJKoZIhvcNAQEB
    BQADggEPADCCAQoCggEBAM3lg3PTMQz4MBgRnsuQKgwcKrSYwXn5A6y0V1qRMxaA
    Ig65g6DZNzRPPNIkHJHikHVmcgvGWOTPiQ6OijwqV9ZaMbyPSg/RriImAcOGLw1P
    JkkIOYZinuhqJNJRdIO7WdslQulXNG8AWFfrP4qmD4+yyLjglwV3TZh1w7TlAroH
    k+Vn0s54Srv9xEMQJDiUODrDoDGqctrTX6k0/oo7fBNeN06fHPHmIK56OzZ4hrZF
    QdjJcV6nbJcenyr2hPOkTy+TojYKn2L7U2BZ2wAKvmZSL8LOChZbzpDbbrJQVKfJ
    tOLOd0kjDnOcXx+RZtCmiDsxztJFH4xC9w7RL5Sc3Q8CAwEAAaAiMCAGCSqGSIb3
    DQEJDjETMBEwDwYDVR0RBAgwBocECgAABDANBgkqhkiG9w0BAQsFAAOCAQEAPSpf
    A9BMiVGnJ1ce0YxVGNAx02WFcbXzF4q8ZryGviSwsUbBkGR56wA1vL/ugbwDxYSG
    IgzpVDgsXfaOkEuYRpxkW0Qn7jVS2E9xs5H+qSB3w5pbiQGirkXpsZCYkZuobeDu
    RzPLLJX/x7biauvy+AijH33QSqw+n7jzsakE8eFXb8WRBZ08DqSRHLhAqRiWnacj
    GSTX1h+srvz85RJsmO74Zq5b78y1Sq6OxMwja4FPaWG6cQ66KRzHdrzZbQotyNF0
    LZ1BG5ZLsvYy8yaSSlxfq1fTYD6ktX4/jfKeTe5uqzrVrNG8zB2meKtlNM5hUpzk
    Uq0n0VvUlll4GJgnRA==
    -----END CERTIFICATE REQUEST-----
  created: 2023-10-24T17:56:38Z
  updated: 2023-10-24T17:56:38Z
  ```

Reply:
It^s added again.